template/template.go

```diff
@@ -58,8 +59,8 @@ func New(options ...Option) (*Template, error) {
 		o(t.text, t.html)
 	}
 
-	t.text.Funcs(tmpltext.FuncMap(DefaultFuncs))
-	t.html.Funcs(tmplhtml.FuncMap(DefaultFuncs))
+	t.text.Funcs(tmpltext.FuncMap(DefaultFuncs)).Funcs(tmpltext.FuncMap(sprig.FuncMap()))
```

Comment:
You need to block functions like [`os`](https://masterminds.github.io/sprig/os.html) so people can't read environment variables.